CHANGELOG.md

```diff
@@ -17,10 +17,13 @@
 - PR #829: C++: enable nvtx ranges
 - PR #835: Holt-Winters algorithm
 - PR #837: treelite for decision forest exchange format
+- PR #840: Dask RF classification and regression
```

Comment:
Why do you have multiple entries in the changelog, and more importantly, why the Dask Random Forest ones?

Reply:
Oh wait I do?????

Reply:
Ohhh I remember. There were merge conflicts, so I copied over the latest changelog.